pages/common/ghci.md

```diff
@@ -0,0 +1,24 @@
+# ghci
+
+> The Glasgow Haskell Compiler's interactive environment.
+> More information: <https://downloads.haskell.org/ghc/latest/docs/html/users_guide/ghci.html>.
+
+- Start an interactive REPL session:
```

Comment:
Based on the `node.md` page:
```suggestion
- Start a REPL (interactive shell):
```

Reply:
That's true, sounds better. I will replace all instances of "an interactive REPS session" with "start a REPL (interactive shell)"